### ISSUE TITLE
Content suggestion: Alt text guidelines

### ISSUE THREAD

**Issue by bsmth** [mdn/content#20948]
## What is the new suggestion?

In our writing guidelines, we should mention some hints for writing alt text.

This could live here -> https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Images_media

Or we could add a new section on Accessibility if there are other related details we're also missing.

## Why is it important or useful?

We would like to add alt text for most (if not all) images on the website. Writing good alt text is hard and we should have some good guidelines on best practices for the content of alt text.

## Other supporting information

>Schalk: Alt text should be a short concise description of the visual asset. You should also not include phrases such as “A photo of,…” as screen readers already does that. The general rule is, if you find that you need more words to describe the image, it is better to use figure with a figcaption or, have a textual description preceding the image. You can then use an empty alt as the image becomes more of a decoration than anything else. You could also, if you really want use aria-describedby to link the descriptive text to the image.

* https://www.w3.org/WAI/tutorials/images/

**Comment by wbamberg** [comment]
It would be great to have clear guidelines for this, but: would they differ from guidance we should have anyway, in the main site rather than the metadocs? If not, and we have this already in the site, why not just link to it. If we don't already have it, why not add it there, and everyone can benefit, not only MDN authors?

(We have some guidance here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt .)

**Comment by teoli2003** [comment]
cc/ @estelle for her opinion
